include/mega/types.h

```diff
@@ -248,16 +248,15 @@ const int FOLDERNODEKEYLENGTH = 16;
 typedef list<class Sync*> sync_list;
 
 // persistent resource cache storage
-struct Cachable
+class Cacheable
```

Comment:
all members are public so struct is more natural than class - why change it?  Save a keyword ;-)

Reply:
A struct shouldn't have virtual functions IMO.